clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/KafkaClusterBasedRecordThrottlerTest.java

```diff
@@ -36,9 +35,6 @@ public void testRecordsCanBeThrottledPerRegion() throws ExecutionException, Inte
     InMemoryKafkaBroker inMemoryRemoteKafkaBroker = new InMemoryKafkaBroker("remote");
     inMemoryRemoteKafkaBroker.createTopic(topic, 2);
 
-    Map<String, Object> extraServerProperties = new HashMap<>();
```

Comment:
Whats up with this code change?

Reply:
super strange behavior of Spotbug. Spotbug didn't detect it before but somehow discover this today 😂

Reply:
ack.  Ok.

Reply:
Thanks for staying vigilant that managers don't start doing code changes, Zac ;P